cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -191,10 +196,25 @@ private CristinObject createObjectWithCristinContributorRoleCode(Integer index,
                 .withEntryCreationDate(LocalDate.now())
                 .withPublicationYear(randomYear())
                 .withContributors(contributors)
+                .withBookReport(randomBookReport())
                 .withPublicationOwner(HardcodedValues.HARDCODED_PUBLICATIONS_OWNER)
                 .build();
     }
 
+    private List<CristinBookReport> randomBookReport() {
+        CristinBookReport bookReport = new CristinBookReport().copy().build();
```

Comment:
Why are you creating an object and then you copy it. The way to use the builder is the following:
```
CristinBookReport bookReport = new CristinBookReport().copy().build();
        CristinBookReport.builder()
            .withIsbn(randomIsbn13())
            .withPublisherName(randomString())
            .withNumberOfPages(randomString());
        return List.of(bookReport);    
```